pkg/daemon/daemon.go

```diff
@@ -233,15 +234,18 @@ func (dn *Daemon) applyNodePTPProfiles() error {
 					if d != nil {
 						time.Sleep(3 * time.Second)
 						go d.cmdRun(false)
-						time.Sleep(2 * time.Second)
+						time.Sleep(5 * time.Second)
+						dn.pluginManager.AfterRunPTPCommand(p.nodeProfile, d.Name())
 						d.monitorEvent(clockType, p.configName, p.exitCh, dn.processManager.eventChannel)
 					}
 				}
 			}
 			time.Sleep(1 * time.Second)
 			go p.cmdRun(dn.stdoutToSocket)
+			dn.pluginManager.AfterRunPTPCommand(p.nodeProfile, p.name)
```

Comment:
do we need to add a few second delay before running this command?

Reply:
We shouldn't.  If necessary for any specific plugin, this can be added inside of the plugin that requires it.